jobs/data/repolist.txt

```diff
@@ -72,6 +72,8 @@ dotnet/roslyn branch=dev15.3-preview1 server=dotnet-ci
 dotnet/roslyn branch=dev15.3-preview1 subFolder=perf definitionScript=perf.groovy server=dotnet-ci2
 dotnet/roslyn branch=dev15.3-preview2 server=dotnet-ci
 dotnet/roslyn branch=dev15.3-preview2 subFolder=perf definitionScript=perf.groovy server=dotnet-ci2
+dotnet/roslyn branch=dev15.3.x server=dotnet-ci
+dotnet/roslyn branch=dev15.3.x subFolder=perf definitionScript=perf.groovy server=dotnet-ci2
 dotnet/roslyn branch=dev15.6 server=dotnet-ci
 dotnet/roslyn branch=dev16 server=dotnet-ci
```

Comment:
I am guessing it is by design that I am not seeing testimpact / LUT branches in this file. But is there some other file I need to modify to enable CI for the dev15.3.x LUT branch? @jasonmalinowski @genlu